README.md

```diff
@@ -40,20 +49,24 @@ Do you want to know more? [Visit the official website](https://adrianmato.com/pe
 The important parts of the project are the following:
 
 ```
-├── .github              GitHub Actions workflows and repo settings
+├── .github                GitHub Actions workflows and repo settings
 ├── src
-│   ├── config.js        Contains the configuration manager
-│   ├── github.js        Business logic: authentication and API calls
-│   ├── ui.js            CLI interactions
-│   └── utils.js         Lightweight utility functions
-├── test                 
-│   └── test.js          Test coverage with Mocha
-├── .prettierrc          Code formatting configuration
-├── index.js             The main thread of execution
-├── README.md            you're looking at it
+│	├── commands
+│   │   ├── codespaces.js  Contains the command to delete codespaces
+│   │   └── repos.js       Contains the command to delete repos
```

Comment:
This indentation doesn't feel right?

Reply:
Oh, seems like there was an issue with tabs and spaces, I have fixed it (https://github.com/adrianmg/github-pewpew/pull/37/commits/8ed64c202ca083d547426a88de1e02ea22068be5)